File: Python/Tools/PasswordTransfer.py
"""
    @Author: isaac
    @Date:   2021-10-27 22:30:29
    @Last Modified by: isaac
    @Last Modified time: 2021-10-27 22:45:29
"""

import json
import base64
import requests
def passwd2Code(code):
    # url = "https://api.jds.codes/jCommand"
    url = str(base64.b64decode('aHR0cHM6Ly9hcGkuamRzLmNvZGVzL2pDb21tYW5k').decode())
    payload = json.dumps({"code": code})
    headers = {'Content-Type': 'application/json'}
    response = requests.request("POST", url, headers=headers, data=payload)
    data = json.loads(response.text)
    print(payload)
    print(data)
    print('\n')
    jumpUrl = data['data']['jumpUrl']
    # print("jumpUrl="+jumpUrl)
    inviteId = jumpUrl.split('inviteId=')[-1].split('&mpin')[0].split('&encryptedPin')[0]
    print("This inviteId is:\n"+inviteId)
if __name__ == "__main__":
    code = input("Please Input The Password:")
    passwd2Code(code)

File: JavaScript/Tools/PasswordTransfer.js
const axios = require('axios');
const code = "12.0:/#FAZ0JFzU9Gb5Qz%"
const data = JSON.stringify({
    "code": code
});
var config = {
    method: 'post',
    url: 'https://api.jds.codes/jd/jcommand',
    headers: {
        'Content-Type': 'application/json'
    },
    data: data
};

axios(config)
    .then((resp) => {
        // console.log(JSON.stringify(resp.data));
        const jumpUrl = resp.data['data']['jumpUrl']
        console.log(jumpUrl)
            // const inviteId = jumpUrl.split('inviteId=')[-1].split('&mpin')[0].split('&encryptedPin')[0]
            // console.log("This inviteId is:\n" + inviteId)
    })
    .catch(function(err) {
        console.log(err);
    });